kroxylicious/src/main/java/io/kroxylicious/proxy/internal/filter/BrokerAddressFilter.java

```diff
@@ -34,11 +34,11 @@ public class BrokerAddressFilter implements MetadataResponseFilter, FindCoordina
 
     private static final Logger LOGGER = LoggerFactory.getLogger(BrokerAddressFilter.class);
 
-    public static class BrokerAddressFilterConfig extends FilterConfig {
+    public static class BrokerAddressBaseConfig extends BaseConfig {
```

Comment:
Not a fan of repeating `Base` in the concrete subclass name. How about just `BrokerAddressConfig`?

Reply:
Yep, bad names, shoulda looked at the IntelliJ rename result more carefully. Pushed fix